### DISCUSSION TITLE
Unable to install infinity offline in virtual machines on the intranet

### DISCUSSION THREAD

**Discussion by just-a-learning-person** [grafana/grafana-infinity-datasource#826] [Q&A]
Grafana version: grafana-enterprise-10.4.2-1
Infinity version: 2.6.2 (yesoreyeram-infinity-datasource-2.6.2.linux_amd64.zip)
I used two different ways to install and failed: 
`grafana-cli --pluginUrl /root/yesoreyeram-infinity-datasource-2.6.2.linux_amd64.zip plugins install yesoreyeram-infinity-datasource`
and 
`unzip yesoreyeram-infinity-datasource-2.6.2.linux_amd64.zip`
`mv yesoreyeram-infinity-datasource /var/lib/grafana/plugins`

Grafana cannot recognize /var/lib/grafana/plugins/yesoreyeram-infinity-datasource
![image](https://github.com/grafana/grafana-infinity-datasource/assets/55468780/fff268c3-9fe1-46f6-9534-8a0648d5f5a7)
![6ad8d595fabbb262031542ec8a4431f](https://github.com/grafana/grafana-infinity-datasource/assets/55468780/161be638-7720-46c6-9790-b2c2f0980306)

**Comment by yesoreyeram** [comment]
Have you tried any other plugin the se way?

**Reply by just-a-learning-person** [reply]
Oops. I only tried the old version of the JSON API. Maybe I should focus on issues with Grafana or the intranet.
